src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.22
+++++++
+* Add windows client support for 'az container exec'.
```

Comment:
This should go under the existing history entry. In fact, you can just merge them since nothing has been released yet.